source/code/plugins/collectd_lib.rb

```diff
@@ -27,7 +27,14 @@ def transform(record, hostname)
       data_items = []
       data_info = {}
       data_info["Timestamp"] = record["Timestamp"]
-      data_info["Host"] = hostname
+
+      host = record["host"] if !record["host"].nil?     
```

Comment:
The automatically generated collectd record has keys starting with lowercase. In the perf blob, we just use the convention that the field names start with a capital letter.